### ISSUE TITLE
Toggle button aria-label attribute value is not configurable

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-combo-box#868]
The `aria-label` currently used is hardcoded directly in template:

https://github.com/vaadin/vaadin-combo-box/blob/7432675574b5395dab5dca9fe99de9fcc8a2447d/src/vaadin-combo-box.html#L66

This is a violation of [AA Criteria of WCAG](https://www.w3.org/TR/WCAG21/#language-of-parts) if used on a page with lang != en. 

The value of the aria-label has to be configurable by the developer for i18n and accessibility.

**Comment by web-padawan** [comment]
Same as https://github.com/vaadin/vaadin-select/issues/218, it would be good to have a common fix in future.
